package.json

```diff
@@ -18,8 +18,7 @@
     "typescript": "4.2.3"
   },
   "peerDependencies": {
-    "@aws-cdk/aws-cloudformation": "^1.91.0",
-    "@aws-cdk/cloudformation-include": "^1.91.0",
-    "@aws-cdk/core": "^1.91.0"
+    "aws-cdk-lib": "2.12.0",
+    "constructs": "10.0.63"
```

Comment:
Would this be a place to use `^` - seeing as downstream consumers may be on a version later than 2.12.0 or 10.0.63?